### ISSUE TITLE
i am unable to get the best answer though i trained many times

### ISSUE THREAD

**Issue by SekharDhana** [bshao001/ChatLearner#60]
hello sir, i am a very beginner for deep learning, i tried your repo now i can able to chat with papaya in command line it is giving perfect answers now what i have done is just replaced the training text files with my own questions and answers and run the bottrainer.py now it is building the epochs now when i test the data it is giving different answers for the same question but not accurate answers i tried tuning the learning rate and mean loss but i dont know how to do that can you please suggest me how to do that.

**Comment by bshao001** [comment]
This is a complicated problem to address. Please refer #58 to get part of the answer. 
There are many factors that can affect the output, not just whether you have predefined an answer/response for a certain question (statement). For example, if you questions predefined are not distinct. Another main reason is the overall data distribution: the attention mechanism normally can suggest an answer for a question (without clearly predefined) as long as the question does not fall way out of the scope of the overall data. Therefore, it cannot be just as simple as you create a pair and place it inside the training data set.

**Comment by sekharBuddha** [comment]
ok, thanks for your answer can you suggest me a bit more on how to do that..?
